### DISCUSSION TITLE
"Kiosk Mode" for TIC-80

### DISCUSSION THREAD

**Discussion by Weeppiko** [nesbox/TIC-80#2582] [Ideas]
It would be good if it is possible to run TIC-80 in a "play only mode", without the editors, since the existing *Dev Mode* is currently just blurred in what people think it is more convenient for them while editing, it doesn't diverges "play" and "edit" into separated contexts.

This "Kiosk Mode" can simply be a command line argument, like `--kiosk` or something, or a console command that enables it until TIC-80 is closed (unless someone have an idea on how to make this permanent and able to be disabled later).

The reason for this suggestion is to *actually* have a way to setup it for people who just want to use it for playing, even if it requires a bit more steps, but people doesn't seems to be able to enter in a consensus about this subject in Dev Mode either (#1950)

**Comment by cetebuk** [comment]
you can start with option --cmd=surf 
( also note [discussion](https://github.com/nesbox/TIC-80/discussions/2575) )

not sure, how to do it on android
